apps/docs/content/_data/components.json

```
@@ -384,16 +384,14 @@
 		"parent": "Tree View"
 	},
 	{
-		"title": "Tag Group",
-		"page": "legacy",
-		"markdown": "./libs/components/src/lib/tag-group/README.md",
-		"children": ["Tag"]
-	},
-	{
-		"title": "Tag",
-		"page": "legacy",
-		"markdown": "./libs/components/src/lib/tag/README.md",
-		"parent": "Tag Group"
+		"title": "Tag & Tag Group",
+		"description": "A Tag Group controls the layout for a collection of Tags. Tags are small, interactive labels used to visually group, classify, or filter criteria.",
```

```suggestion
		"description": "Tags are small, interactive labels used to visually group, classify, or filter criteria. Tags are wrapped with Tag Group",
```
I think that tags are more "important" and tag-group is here for semantics and a11y.